app/pages/onboarding/03-restore-wallet/restore-wallet.tsx

```diff
@@ -31,18 +31,13 @@ export const RestoreWallet: React.FC = () => {
     setMnemonic(e.currentTarget.value.trim());
   };
 
+  const mnemonicLength = mnemonic.trim().split(' ').length;
+
   const handleSecretKeyRestore = (e: React.FormEvent) => {
     e.preventDefault();
 
-    const mnemonicLength = mnemonic.trim().split(' ').length;
-    if (mnemonicLength === 12) {
-      setError(
-        'You’ve entered a 12-word Secret Key. Please enter the 24-word Secret Key you received when creating this wallet.'
-      );
-      return;
-    }
-    if (mnemonicLength !== 24) {
-      setError('The Stacks Wallet can only be used with 24-word Secret Keys');
+    if (mnemonicLength !== 12 && mnemonicLength !== 24) {
+      setError('The Stacks Wallet can only be used with 12 and 24-word Secret Keys');
```

Comment:
Super grammatical nitpik: "The Stacks Wallet can be used with _only_ 12 and 24-word Secret Keys"

Reply:
This doesn't sound like correct English to me.

Reply:
Haha this basically:

![image](https://user-images.githubusercontent.com/28991/100094765-cb237e00-2e59-11eb-82b9-4851c57fbab8.png)

Reply:
But the word only in this context is modifying the verb use

![image](https://user-images.githubusercontent.com/1618764/100095410-b5fb1f00-2e5a-11eb-9f4b-6aca718c031c.png)

Reply:
It's modifying the key nouns – to the exclusion of other types of keys (e.g. 8-word keys). 

If it were modifying the verb phrase "can be used", then it would imply, for example, that "the Stacks Wallet can _not_ be used with Linux, Windows or MacOS" (which is obviously not true). The Stacks Wallet can be used in many ways, with or without respect to the key type.

Reply:
John doesn't cook meat for anyone but his family. I don't like dancing outside of parties. Etc.

Reply:
I've changed it, but for the record my Mum also thinks the 'only' should come first 😂

Reply:
To be fair, being grammatically correct doesn't mean sounding the best, so you and your Mum have a point there 😉